jest.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   testEnvironment: 'node',
-  projects: ['<rootDir>/packages/*'],
-  roots: ['packages'],
+  roots: ['commands', 'lib'],
   collectCoverage: true,
+  testPathIgnorePatterns: ['commands/functions/test.js'],
```

Comment:
Is this intentionally here? Or just leftover from a test?

Reply:
This is unchanged from the existing jest.config.js. But from what I can tell, it's not actually doing anything